perun-core/src/main/resources/oracleChangelog.txt

```diff
@@ -72,7 +82,7 @@ alter table USERS modify created_by nvarchar2(1300);
 alter table USER_ATTR_VALUES modify created_by nvarchar2(1300);
 alter table USER_EXT_SOURCES modify created_by nvarchar2(1300);
 alter table USER_FACILITY_ATTR_VALUES modify created_by nvarchar2(1300);
-alter table USER_EXT_SOURCE_ATTR_VALUES modify created_by nvarchar2(1300);
+alter_table USER_EXT_SOURCE_ATTR_VALUES modify created_by nvarchar2(1300);
```

Comment:
Please remove ``_`` in ``alter_table`` (twice). It was fixed after you rebased, so maybe rebase on master one more time or fix it manually. Thank you.